launchable/commands/subset.py

```diff
@@ -338,8 +345,8 @@ def run(self):
 
                     original_subset = res.json().get("testPaths", [])
                     original_rests = res.json().get("rest", [])
-                    subset_id = res.json()["subsettingId"]
-                    summary = res.json()["summary"]
+                    subset_id = res.json().get("subsettingId", 0)
+                    summary = res.json().get("summary", {})
```

Comment:
There won’t be the such case that the summary is empty. But if the summary is empty and called by an old format, it will error. so I fixed this.